app/modules/zoo-api/zooapi.factory.js

```diff
@@ -4,10 +4,21 @@ require('./zooapi.module.js')
     .factory('zooAPI', zooAPI);
 
 var ApiClient = require('panoptes-client/lib/api-client');
-
+var OAuth = require('panoptes-client/lib/oauth')
 // @ngInject
-function zooAPI(zooAPIConfig) {
-
+function zooAPI($rootScope, zooAPIConfig) {
```

Comment:
Not a blocking comment, but what does `zooAPIConfig` do? I don't see it anywhere else in this function.

Reply:
Good point. Seems to be doing nothing.